src/pages/Admin/AdminMainLayout.tsx

```diff
@@ -35,14 +36,17 @@ export const AdminMainLayout = () => {
     fetchAdminCohorts();
   }, [fetchAdminCohorts]);
 
-  return cohortsList?.length ? (
+  // return cohortsList?.length ? (
+  return (
     <div>
       <Container>
         <AdminNavbar />
+        <BreadcrumbMenu />
         <Outlet context={{ cohorts: cohortsList }} />
```

Comment:
you probably need to wrap them with a Flex so that they are in the proper layout